docs/usage/03-sqon-filter-notation.md

```diff
@@ -0,0 +1,107 @@
+# SQON Filter Notation
+
+<!--To understand SQONs better we should explain why they are important and useful in arranger-->
+
+Arranger uses a custom filter notation SQON (Serializable Query Object Notation, pronounced like "Scone") to provide a flexible system for combining filters in a JSON object format.
+
+- A SQON object consists of two types of nested objects, **Value Objects** and **Operation Objects**.
```

Comment:
I think using "object" so repeatedly will quickly lead to semantic satiation, and that we can rephrase this later on using other more descriptive terms like "entity" and "type"